tests/prague/eip7692_eof_v1/eip7069_extcall/test_calls.py

```diff
@@ -318,6 +321,71 @@ def test_eof_calls_eof_mstore(
     )
 
 
+identity = Address(0x04)
+blake2f = Address(0x09)
+
+
+@pytest.mark.parametrize(
+    "opcode",
+    [
+        Op.EXTCALL,
+        Op.EXTDELEGATECALL,
+        Op.EXTSTATICCALL,
+    ],
+)
+@pytest.mark.parametrize(
+    "precompile", [pytest.param(identity, id="identity"), pytest.param(blake2f, id="blake2f")]
+)
+def test_eof_calls_precompile(
+    state_test: StateTestFiller,
+    pre: Alloc,
+    sender: EOA,
+    opcode: Op,
+    precompile: Address,
+):
+    """Test EOF contracts calling precompiles"""
+    env = Environment()
+
+    caller_contract = Container.Code(
+        Op.MSTORE(0, value_returndata_magic)
+        + Op.SSTORE(slot_call_result, opcode(address=precompile, args_offset=0, args_size=32))
+        + Op.SSTORE(slot_returndatasize, Op.RETURNDATASIZE)
+        + Op.SSTORE(slot_returndata, Op.RETURNDATALOAD(0))
+        + Op.SSTORE(slot_code_worked, value_code_worked)
+        + Op.STOP,
+    )
+    calling_contract_address = pre.deploy_contract(caller_contract)
+
+    tx = Transaction(
+        sender=sender,
+        to=Address(calling_contract_address),
+        gas_limit=50000000,
```

Comment:
Lower gas limit of possible.

Reply:
Done, but not too much. The inner call consumes all (63/64th) gas and the 1/64th must be enough for a bunch of sstore's :shrug: